pypeit/core/load.py

```diff
@@ -75,7 +75,7 @@ def load_ordloc(fname):
     return ltrace, rtrace
 
 
-def load_specobjs(fname,order=None):
+def load_specobjs(fname,order=None, verbose=False):
```

Comment:
Yes I agree, I've been begging for someone to revamp specobjs for quite some time. For now I'm going to leave this as is, since I think our data output model needs to undergo serious changes.